exercises/leap/cases_test.go

```diff
@@ -1,7 +1,7 @@
 package leap
 
 // Source: exercism/x-common
-// Commit: be6fa53 leap: Rewrite the test cases and their descriptions
+// x-common version: 1.0.0
```

Comment:
If you wonder why I linked to Crystal, it's just for reference to see if you can get any ideas from them.